pkg/compactor/split_merge_compactor_test.go

```diff
@@ -142,10 +142,6 @@ func TestMultitenantCompactor_ShouldSupportSplitAndMergeCompactor(t *testing.T)
 				block1 := createTSDBBlock(t, bkt, userID, 0, (5 * time.Minute).Milliseconds(), numSeries, externalLabels(""))
 				block2 := createTSDBBlock(t, bkt, userID, time.Minute.Milliseconds(), (7 * time.Minute).Milliseconds(), numSeries, externalLabels(""))
 
-				// Add another block as "most recent one" otherwise the previous blocks are not compacted
```

Comment:
These additional blocks are no longer needed since the new split_merge_grouper conditions allow compaction to proceed based on the job's time being < now.